### PR TITLE
feat(Dockerfile): add helm and kubectl

### PR DESCRIPTION
**Pull request by mboersma**
This enables the container to be used as a more complete development environment, which will help us with doing some tasks on Windows in particular.

### CODE REVIEW

File: rootfs/Dockerfile [hunk 1]
```diff
@@ -29,6 +31,9 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
   && curl -sSL -o /tmp/protoc.zip https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip \
   && unzip /tmp/protoc.zip 'bin/protoc' -d /usr/local \
   && rm /tmp/protoc.zip \
+  && curl -sSL https://storage.googleapis.com/kubernetes-release/release/$KUBECTL_VERSION/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl \
+  && chmod +x /usr/local/bin/kubectl \
+  && curl -sSL https://storage.googleapis.com/kubernetes-helm/helm-$HELM_VERSION-linux-amd64.tar.gz | tar -C /usr/local -xz \
```
Comment (ultimateboy) [inline rootfs/Dockerfile]: can you rm the tar after this please?

Comment (jchauncey) [inline rootfs/Dockerfile]: if you pipe it to tar does it actually ever go to disk?

Comment (ultimateboy) [inline rootfs/Dockerfile]: Oh, you are totally right. /me didnt scroll far enough to the right to see this is being piped directly to tar. This should be fine

Comment (mboersma) [inline rootfs/Dockerfile]: Oops, forgot to save my buffer. I pushed a small update to this line, everything's happy now.

**Pull request merged**